### PR TITLE
Revise tutorial copy

### PR DESCRIPTION
**Pull request by ampinsk**
This updates the copy for the tutorial to hopefully add some more clarity. Would love some more eyes on this! This should be merged before our next release to beta. The copy changes are in [`app/src/ui/tutorial-panel/tutorial-panel.tsx`](https://github.com/desktop/desktop/pull/8294/files#diff-5bfaade88db12d59f743b517f204867f)

~~⚠️ This should be merged after #8242!~~

cc @outofambit @billygriffin @nerdneha

### CODE REVIEW

File: app/src/ui/tutorial-panel/tutorial-panel.tsx [hunk 1]
```diff
@@ -87,37 +121,40 @@ export class TutorialPanel extends React.Component<
...
+            isComplete={this.isStepComplete}
+            isNextStepTodo={this.isStepNextTodo}
+            sectionId={TutorialStep.CreateBranch}
             currentlyOpenSectionId={this.state.currentlyOpenSectionId}
-            onClick={this.handleToggle}
+            onSummaryClick={this.onStepSummaryClick}
           >
             <p className="description">
-              {`Create a branch by going into the branch menu in the top bar and
+              {`Branches are how you work on different versions of a repository at one time. Create a
```
Comment (billygriffin) [inline app/src/ui/tutorial-panel/tutorial-panel.tsx]:
```suggestion
              {`A branch allows you to work on different versions of a repository at one time. Create a
```

File: app/src/ui/tutorial-panel/tutorial-panel.tsx [hunk 2]
```diff
@@ -87,37 +121,40 @@ export class TutorialPanel extends React.Component<
...
+            {!this.isStepComplete(TutorialStep.PickEditor) && (
+              <div className="action">
+                <LinkButton onClick={this.skipEditorInstall}>
+                  I have an editor
+                </LinkButton>
+              </div>
+            )}
+          </TutorialStepInstructions>
+          <TutorialStepInstructions
             summaryText="Make a branch"
```
Comment (billygriffin) [inline app/src/ui/tutorial-panel/tutorial-panel.tsx]:
```suggestion
            summaryText="Create a branch"
```

File: app/src/ui/tutorial-panel/tutorial-panel.tsx [hunk 3]
```diff
@@ -126,67 +163,69 @@ export class TutorialPanel extends React.Component<
...
+            isNextStepTodo={this.isStepNextTodo}
+            sectionId={TutorialStep.MakeCommit}
             currentlyOpenSectionId={this.state.currentlyOpenSectionId}
-            onClick={this.handleToggle}
+            onSummaryClick={this.onStepSummaryClick}
           >
             <p className="description">
-              Write a message that describes the changes you made. When you’re
-              done, click the commit button to finish.
+              Commits are how you save sets of changes. Write a
```
Comment (billygriffin) [inline app/src/ui/tutorial-panel/tutorial-panel.tsx]:
```suggestion
              A commit allows you to save sets of discrete changes. Write a
```

File: app/src/ui/tutorial-panel/tutorial-panel.tsx [hunk 4]
```diff
@@ -126,67 +163,69 @@ export class TutorialPanel extends React.Component<
...
+            isComplete={this.isStepComplete}
+            isNextStepTodo={this.isStepNextTodo}
+            sectionId={TutorialStep.OpenPullRequest}
             currentlyOpenSectionId={this.state.currentlyOpenSectionId}
-            onClick={this.handleToggle}
+            skipLinkButton={<SkipLinkButton onClick={this.skipCreatePR} />}
+            onSummaryClick={this.onStepSummaryClick}
           >
             <p className="description">
               Pull Requests are how you propose changes. By opening one, you’re
```
Comment (billygriffin) [inline app/src/ui/tutorial-panel/tutorial-panel.tsx]:
```suggestion
              A pull requests allows you to propose changes to the code. By opening one, you’re
```

File: app/src/ui/tutorial-panel/tutorial-panel.tsx [hunk 5]
```diff
@@ -126,67 +163,68 @@ export class TutorialPanel extends React.Component<
...
+            isNextStepTodo={this.isStepNextTodo}
+            sectionId={TutorialStep.MakeCommit}
             currentlyOpenSectionId={this.state.currentlyOpenSectionId}
-            onClick={this.handleToggle}
+            onSummaryClick={this.onStepSummaryClick}
           >
             <p className="description">
-              Write a message that describes the changes you made. When you’re
-              done, click the commit button to finish.
+              Commits are how you save sets of changes. Write a message that
```
Comment (billygriffin) [inline app/src/ui/tutorial-panel/tutorial-panel.tsx]:
```suggestion
              A commit allows you to save sets of changes. Write a message that
```

**Comment by tierninho** [comment]
The changes are overall good, however these two sentences had repeating prepositions and it seemed a bit choppy:

> Write a message that describes the changes you made in the text box in the bottom left area.

and 

> Publishing will “push”, or upload, your commits to this branch on this repository on GitHub.com.

Suggestions:

1.  In the bottom-left text area, write a message that describes the changes you made and click the blue Commit button to finish.
2. Publishing will “push”, or upload, your commits to this branch on this Github.com repository.

**Comment by niik** [comment]
Gonna merge `epic/tutorial` to this branch to make this easier to review now that #8242 has been merged.

**Pull request merged**